File: app/src/main/java/libs/com/rogerio/utils/Sounds.java
package libs.com.rogerio.utils;

import android.content.Context;
import android.media.AudioManager;

public class Sounds {
	protected float leftVolume;
	protected float rightVolume;
	AudioManager manager;	
	
	public Sounds(Context ctx) {
		manager =(AudioManager) ctx.getSystemService(Context.AUDIO_SERVICE);
	}

	protected void volume() {
		float curVolume = manager.getStreamVolume(AudioManager.STREAM_MUSIC);
		float maxVolume = manager.getStreamMaxVolume(AudioManager.STREAM_MUSIC);
		leftVolume = curVolume/maxVolume;
		rightVolume = curVolume/maxVolume;
	}

}

File: app/src/main/java/libs/com/rogerio/utils/Sounds.kt
package libs.com.rogerio.utils

import android.content.Context
import android.media.AudioManager

open class Sounds(ctx: Context) {
    protected var leftVolume: Float = 0.toFloat()
    protected var rightVolume: Float = 0.toFloat()
    internal var manager: AudioManager

    init {
        manager = ctx.getSystemService(Context.AUDIO_SERVICE) as AudioManager
    }

    protected fun volume() {
        val curVolume = manager.getStreamVolume(AudioManager.STREAM_MUSIC).toFloat()
        val maxVolume = manager.getStreamMaxVolume(AudioManager.STREAM_MUSIC).toFloat()
        leftVolume = curVolume / maxVolume
        rightVolume = curVolume / maxVolume
    }

}